AppIntegrity/Sources/AppIntegrity/URLRequest+Integrity.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+extension URLRequest {
+    static func clientAttestation(token: String) throws -> URLRequest {
+        var components = URLComponents()
+        components.scheme = "https"
+        components.host = "app-integrity-spike.mobile.dev.account.gov.uk"
```

Comment:
Is there a plan to get our environment base urls in here?

Reply:
Yes, it's gonna be the same format as the `MobilePlatformServices` package uses 🙂